kivy/uix/screenmanager.py

```diff
@@ -1068,11 +1069,16 @@ def on_current(self, instance, value):
             screen.dispatch('on_pre_enter')
             screen.dispatch('on_enter')
 
+    def get_screens_with_name(self, name):
```

Comment:
We want to expose that and not to keep it for internal use?

How about `_get_screens_with_name` ?

Reply:
We really are using this function internally only, so it is better the make the suggested change. About the tests, I'll do it in a few days!